common-ui/src/main/res/values/attrs_lists.xml

```diff
@@ -16,17 +16,18 @@
 
 <resources>
 
-    <!--Common lists attributes-->
+    <!--Default lists attributes-->
     <attr name="listItemTitleColor" format="color"/>
     <attr name="listItemSubtitleColor" format="color"/>
```

Comment:
NIT: Maybe we could update these attribute names to `listItemPrimaryTextColor` and `listItemSecondaryTextColor` for consistency.

Reply:
good idea!